observability/opentelemetry/src/main/java/org/wildfly/extension/opentelemetry/OpenTelemetrySubsystemRegistrar.java

```diff
@@ -200,6 +206,19 @@ public ManagementResourceRegistration register(SubsystemRegistration parent,
 
     @Override
     public ResourceServiceInstaller configure(OperationContext context, ModelNode model) throws OperationFailedException {
+        List<String> otherMetrics = new ArrayList<>();
+        if (context.getCapabilityServiceSupport().hasCapability(CAPABILITY_NAME_METRICS)) {
+            otherMetrics.add("WildFly Metrics");
+        }
+        if (context.getCapabilityServiceSupport().hasCapability(CAPABILITY_NAME_MICROMETER)) {
+            otherMetrics.add("Micrometer");
+        }
+        if (!otherMetrics.isEmpty()) {
+            if (Boolean.parseBoolean(System.getProperty("wildfly.multiple.metrics.warn", "true"))) {
+                OTEL_LOGGER.multipleMetricsSystemsEnabled(String.join(",", otherMetrics));
+            }
+        }
```

Comment:
This is a tangent to this PR, sparked by my thinking about the problem you're solving here.

https://opentelemetry.io/docs/languages/java/configuration/#properties-exporters lists 'none' as a valid value for otel.metrics|traces|logs.exporter, which implies that in the future we could perhaps allow the disabling any of those via the subsystem API. With otel.metrics.exporter=none being the one that's relevant to the problem you're addressing here.

Reply:
I've thought of that, too, but it gets complicated as that's "just" a system property for OpenTelemetry, but, for MicroProfile Telemetry, it would have to come from a MicroProfile ConfigSource to be exhaustive, and it didn't seem prudent (or currently possible) to do a check like that in the subsystem code.

Reply:
And we could actually add those now via the in-flight WFLY-20296...